bevy_lint/tests/ui/missing_reflect/impl.rs

```diff
@@ -42,5 +48,17 @@ impl Component for MyEvent {
 
 //~v NOTE: `Event` implemented here
 impl Event for MyEvent {
-    type Traversal = ();
+    type Trigger<'a> = GlobalTrigger;
+}
+
+//~| HELP: `Reflect` can be automatically derived
+//~v ERROR: defined a message without a `Reflect` implementation
+struct MyMessage(String);
+
+impl Component for MyMessage {
+    const STORAGE_TYPE: StorageType = StorageType::Table;
+    type Mutability = Mutable;
 }
```

Comment:
I don't think this is needed anymore, since `Message` no longer requires `Component`.